commands/execenv/loading.go

```
@@ -171,3 +172,15 @@ func CacheBuildProgressBar(env *Env, events chan cache.BuildEvent) error {
 
 	return nil
 }
+
+func getRepoPath(env *Env) (string, error) {
```

`getRepoPath` should return an error if the path doesn't exist.